src/tox.ini

```diff
@@ -16,38 +16,29 @@ install_command =
   pip install {opts} {packages}
 
 [testenv:pep8]
-basepython = python2.7
+basepython = python3
 commands = charm-proof
 
-[testenv:func27-noop]
-# DRY RUN - For Debug
-basepython = python2.7
-commands =
-    bundletester -vl DEBUG -r json -o func-results.json --test-pattern "gate-*" -n --no-destroy
+[testenv:venv]
+basepython = python3
+commands = {posargs}
 
-[testenv:func27]
-# Run all gate tests which are +x (expected to always pass)
-basepython = python2.7
+[testenv:func-noop]
+basepython = python3
 commands =
-    bundletester -vl DEBUG -r json -o func-results.json --test-pattern "gate-*" --no-destroy
+    true
```

Comment:
The charm is to be Python3.  Please remove framework for testing Python 2.7.

Reply:
Hi Ryan, thanks for your comments. I may be wrong or missing something, but I don't see something related to python2.7